front/app/modules/commercial/native_surveys/admin/components/SurveyBuilderTopBar/index.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+
+// hooks
+import useProject from 'hooks/useProject';
+import useLocalize from 'hooks/useLocalize';
+
+// components
+import GoBackButton from 'components/UI/GoBackButton';
+import Button from 'components/UI/Button';
+
+// styling
+import { colors } from 'utils/styleUtils';
+import {
+  Box,
+  stylingConsts,
+  Spinner,
+  Text,
+  Title,
+} from '@citizenlab/cl2-component-library';
+
+// utils
+import { isNilOrError } from 'utils/helperUtils';
+
+// i18n
+import messages from './messages';
+import { FormattedMessage } from 'utils/cl-intl';
+
+// routing
+import clHistory from 'utils/cl-router/history';
+import { withRouter, WithRouterProps } from 'utils/cl-router/withRouter';
+
+const SurveyBuilderTopBar = ({ params: { projectId } }: WithRouterProps) => {
+  const localize = useLocalize();
+  const project = useProject({ projectId });
+
+  const goBack = () => {
+    clHistory.push(`/admin/projects/${projectId}/native-survey`);
+  };
+
+  return (
+    <Box
+      position="fixed"
+      zIndex="3"
+      alignItems="center"
+      w="100%"
+      h={`${stylingConsts.menuHeight}px`}
+      display="flex"
+      background={`${colors.adminContentBackground}`}
+      borderBottom={`1px solid ${colors.mediumGrey}`}
+    >
+      <Box
+        p="15px"
+        w="210px"
+        h="100%"
+        borderRight={`1px solid ${colors.mediumGrey}`}
+        display="flex"
+        alignItems="center"
+      >
+        <GoBackButton onClick={goBack} />
+      </Box>
+      <Box display="flex" p="15px" flexGrow={1} alignItems="center">
```

Comment:
Update to 16px for the padding?

Reply:
@amanda-anderson for the styles, I was trying to keep it consistent with what we have in the content builder. I didn't want to have many changes there. Do you think it makes sense to them there as-well?

Reply:
Hmm good point.. I don't think they were caught in the content builder. Whatever you think makes the most sense then :+1: :)

Reply:
🤔 Let me make the changes here so we don't make the mistake twice